packages/controller-utils/CHANGELOG.md

```diff
@@ -51,30 +51,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Remove the entry for localhost from `BUILT_IN_NETWORKS`, `NetworkType`, `ChainId`, and `NetworksTicker`
 - **BREAKING:** Remove `hasProperty` function ([#1275](https://github.com/MetaMask/core/pull/1275))
   - Use the `hasProperty` function from `@metamask/utils` instead
+- **BREAKING:** Remove constants `MAINNET` and `TESTNET_TICKER_SYMBOLS` ([#1132](https://github.com/MetaMask/core/pull/1132))
```

Comment:
I have gone back to update the GitHub release page for v3.1.0 and v4 with these details. I didn't add the "DEPRECATED" status because it was just confusing without the package version present.

I have marked these packages as deprecated on npm as well